pkg/start/start.go

```diff
@@ -87,6 +92,7 @@ func NewOptions() *Options {
 		PayloadOverride: os.Getenv("PAYLOAD_OVERRIDE"),
 		ResyncInterval:  minResyncPeriod,
 		EnableMetrics:   true,
+		Exclude:         os.Getenv("EXCLUDE_MANIFESTS"),
```

Comment:
When exclude is set we should log this.

`Excluding manifests for %q`

in the main logs so we can see it if we have bugs.

Reply:
done